frontend/apps/crates/components/src/page_header/dom.rs

```diff
@@ -55,6 +56,11 @@ pub fn render(
                 })
                 .property("href", &page_link.route())
                 .property("target", page_link.target())
+                .event(move |_evt: events::Click| {
+                    let mut properties = HashMap::new();
+                    properties.insert("Header Kind", page_link.kind_str().to_owned());
+                    mixpanel::track("Header Click", Some(properties));
+                })
```

Comment:
Are we going to have to call a function for each and every event to get mixpanel to work?

Reply:
This is similar to how we would do it in JS: `mixpanel.track('something', {prop: 'value'})`

Reply:
But I'm guessing it's going to be in each event handler?